### ISSUE TITLE
Build riscv qemu port

### ISSUE THREAD

**Issue by shlevy** [NixOS/nixpkgs#35087]

**Comment by castilma** [comment]
I installed qemu-2.11.1 through nix-env -iA nixos.qemu. But now, when I `nix-env -u` I see 
```upgrading 'qemu-2.11.1' to 'qemu-2.11.50pre58771_af435b709d'
these paths will be fetched (6.70 MiB download, 39.56 MiB unpacked):
  /nix/store/h0qv6i5qaxv43g3cnbfl0mlzlz51k1g7-qemu-2.11.50pre58771_af435b709d
copying path '/nix/store/h0qv6i5qaxv43g3cnbfl0mlzlz51k1g7-qemu-2.11.50pre58771_af435b709d' from 'https://cache.nixos.org'...
building '/nix/store/bvhs4jjxla8x9jp91g07hk3m6albi51y-user-environment.drv'...
created 7117 symlinks in user environment
```
This replaces qemu-system-x86_64 and every other arch for riscv. is that wanted behaviour?
 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.32, NixOS, 18.03.131954.2569e482904 (Impala)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0`
 - channels(mcd): `""`
 - channels(root): `"nixos-18.03.131954.2569e482904, nixos-1709-17.09.3238.e984f9e48e1"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by shlevy** [comment]
@castilma Will fix, thanks.

**Comment by shlevy** [comment]
@castilma Fixed in git master and release-18.03.

**Comment by shlevy** [comment]
As a work around, you can do `nix-env -iA nixos.qemu` to upgrade, until the channels update with my fix.
